maven/lib/dependabot/maven/update_checker/requirements_updater.rb

```diff
@@ -50,12 +50,8 @@ def updated_requirements
         attr_reader :properties_to_update
 
         def update_requirement(req_string)
-          if req_string.include?(".+")
-            update_dynamic_requirement(req_string)
-          else
-            # Since range requirements are excluded this must be exact
-            update_exact_requirement(req_string)
-          end
+          # Since range requirements are excluded this must be exact
```

Comment:
There's no special treatment of `+` in the maven spec here: https://maven.apache.org/pom.html#version-order-specification so my presumption is that this is legacy code.